File: Programmers/level1/intervalX/interval_x.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12954

def solution(x, n):
    answer = [x]
    while n > 1:
        answer.append(answer[-1] + x)
        n -= 1
    print(answer)
    return answer

# def solution(x, n):
#     return [(i*x)+x for i in range(n)]

File: Programmers/level1/intervalX/intervalX.js
// https://programmers.co.kr/learn/courses/30/lessons/12954?language=javascript

export const solution = (x, n) => {
    let answer = [];
    for(let i = 1; i < n+1; i++){
        answer.push(x*i)
    }
    return answer;
}